src/proper.erl

```diff
@@ -1388,27 +1389,29 @@ run(Result, #ctx{mode = Mode, bound = Bound} = Ctx, _Opts) when is_boolean(Resul
 	false ->
 	    {error, too_many_instances}
     end;
-run({exists, TMap, Prop, Not}, #ctx{mode = new} = Ctx,
+run({exists, RawType, Prop, Not}, #ctx{mode = new} = Ctx,
     #opts{search_strategy = Strat, search_steps = Steps,
           output_fun = Print, start_size = StartSize} = Opts) ->
-    proper_target:init_strategy(Strat),
-    Target = proper_target:targeted(TMap),
-    Print("[", []),
+    proper_target:init_strategy(#{search_strategy => Strat,
```

Comment:
Since you are changing this code, this is better written in two lines as:
```erlang
    InitOpts = #{search_strategy => Strat, search_steps => Steps}),
    proper_target:init_strategy(InitOpts),
```